WordPress/src/main/java/org/wordpress/android/viewmodel/pages/PagesViewModel.kt

```diff
@@ -680,12 +686,10 @@ class PagesViewModel
     private fun hasRemoteAutoSavePreviewError() = _previewState.value != null &&
             _previewState.value == PostListRemotePreviewState.REMOTE_AUTO_SAVE_PREVIEW_ERROR
 
-    @Subscribe(threadMode = ThreadMode.MAIN)
-    @SuppressWarnings("unused")
-    fun onPostUploaded(event: OnPostUploaded) {
+    fun handlePostUploadedWithoutError(remotePostId: RemoteId) {
         var id = 0L
         if (!pageUpdateContinuations.contains(id)) {
```

Comment:
Honestly, no it doesn't.  I just migrated and left it because I thought it was outside the scope of the refactor but thanks for pointing it out. I am going to investigate why it was introduced or what the purpose of it is supposed to be 🙏

Reply:
I agree it's out-side of scope. But if the logic doesn't make sense, I'd consider creating an issue.

Reply:
Thanks! Good idea https://github.com/wordpress-mobile/WordPress-Android/issues/11244